### DISCUSSION TITLE
RPI 1 Difficulties

### DISCUSSION THREAD

**Discussion by Mohoganogan** [rern/rAudio#42] [Fixed Issues]
Using a Raspberry Pi 1 B

Flash card
boot card while attached to screen
normal boot type text scrolls past
screen clears, rAudio login:
then a series of audit: type 11-- messages
no on screen message similar to what I have seen with my Pi3
going to RPI address via browser does eventually (very slowly) present the web interface.

**Comment by rern** [comment]
There's no Browser on RPi for RPi Zero and 1. It's too much for the CPU.
